packages/components/organisms/f-header/src/components/_tests/__snapshots__/Header.test.js.snap

```diff
@@ -1,9 +1,9 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Header should render default component markup 1`] = `
-<header data-theme="je" data-test-id="header-component" class="c-header c-header--transparent c-header--gradient">
+<header data-theme="je" data-test-id="header-component" class="undefined undefined">
```

Comment:
@FayeCarter This doesn't quite look right to me

Reply:
I think it's cos jest doesn't recognise CSS modules, but I will definitely triple check. Looks very dodgy 😬

Reply:
@xander-marjoram I've added in a `$style` mock to the tests. It seems happier now 😄, thanks!